news/_posts/2023-10-12-sc_subdomain.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Single cell subdomain re-launch: Unified and feedback-driven"
+contributions:
+  authorship: [nomadscientist, pavanvidem, "José Manuel Domínguez"]
+tags: [gtn, singlecell]
+cover: "news/images/2023-10-12-singlecell_subdomain.png"
+layout: news
+---
+
+The European Galaxy Days 2023 CoFest combined the forces of administrator, developer and trainer to update and re-launch the [single cell](https://singlecell.usegalaxy.eu) Galaxy instance. Where previously there were two subdomains each with their own sets of tools, there is now a single subdomain with a suite of tools in an order that makes sense for users.
```

Comment:
```suggestion
The European Galaxy Days 2023 CoFest combined the forces of administrator, developer and trainer to update and re-launch the [single cell](https://singlecell.usegalaxy.eu) Galaxy instance. Where previously there were two subdomains each with their own sets of tools, there is now a unified subdomain with re-categorized tools that makes sense for users.
```

Reply:
hmm I think 

> there is now a single subdomain with a suite of tools in an order that makes sense for users.

is just fine. I don't think it needs to be changed.